templates/visuals/default/src/visual.ts

```diff
@@ -37,6 +37,7 @@ module powerbi.extensibility.visual {
         }
 
         public update(options: VisualUpdateOptions) {
+            this.settings = Visual.parseSettings(options.dataViews[0]);
```

Comment:
Please consider that `options` and `options.dataViews` might be undefined. It would throw an exception.